specification/iotcentral/resource-manager/Microsoft.IoTCentral/preview/2021-11-01-preview/iotcentral.json

```diff
@@ -0,0 +1,1259 @@
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "This is a synchronous operation. The body contains a JSON-serialized array of the metadata from all the Private Endpoint Connections of the IoT Central Application.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v4/privatelinks.json#/definitions/PrivateEndpointConnectionListResult"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.IoTCentral/iotApps/{resourceName}/privateLinkResources/{privateLinkName}": {
+      "get": {
+        "tags": [
+          "Networking"
+        ],
+        "description": "Get a Private Link of a IoT Central Application.",
+        "operationId": "PrivateLinks_Get",
+        "x-ms-examples": {
+          "PrivateLinks_Get": {
+            "$ref": "./examples/PrivateLinks_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "name": "privateLinkName",
+            "in": "path",
+            "description": "The private link resource name.",
+            "required": true,
+            "type": "string",
+            "x-ms-parameter-location": "method"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The private link resource for the IoT Central Application.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v4/privatelinks.json#/definitions/PrivateLinkResource"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.IoTCentral/iotApps/{resourceName}/privateLinkResources": {
+      "get": {
+        "tags": [
+          "Networking"
+        ],
+        "description": "Get all Private Links of a IoT Central Application.",
```

Comment:
Can we confirm with the naming on these GET API calls for retrieving the Private Link Resources for an Azure resource. "Private Link" seems odd to me but if this is what other providers are doing we can leave as is.

Reply:
@r-delgadillo "private link resource(s)" is common. the parameter itself is most often called "groupId", "groupName", and "privateLinkResourceName"? do you prefer any one of these?

Reply:
Here is what we currently have in RP service: 

/subscriptions/:subscriptionId/resourceGroups/:resourceGroupName/providers/:resourceProviderNamespace/:resourceType/:resourceName/privateLinkResources/:groupId

I'm ok with changing this to the following if it's common across different RPs

/subscriptions/:subscriptionId/resourceGroups/:resourceGroupName/providers/:resourceProviderNamespace/:resourceType/:resourceName/privateLinkResources/:privateLinkResourceName

Reply:
"groupId" is a little more common so i'll move to that